### DISCUSSION TITLE
Code line numbers for Docusaurus

### DISCUSSION THREAD

**Discussion by eitsupi** [quarto-dev/quarto-cli#5139] [Feature Requests]
Quarto supports the display of line numbers in code blocks in HTLM format, etc.
https://quarto.org/docs/output-formats/html-code.html

The syntax is thus.

````qmd
```{.python code-line-numbers="true"}
import matplotlib.pyplot as plt
plt.plot([1,23,2,4])
plt.show()
```
````

The syntax in Docusaurus seems to be as follows, ` showLineNumbers` is needed.
https://docusaurus.io/docs/markdown-features/code-blocks#line-numbering

````md
```python showLineNumbers
import matplotlib.pyplot as plt
plt.plot([1,23,2,4])
plt.show()
```
````

It would be great if Quarto could support this conversion.

**Comment by cderv** [comment]
Yes I think this should be supported. I transformed to issue #5152 to track this ! Thanks

**Reply by eitsupi** [reply]
Thanks, I close this.

**Comment by jjallaire** [comment]
We'll support this in v1.4 w/ this change: https://github.com/quarto-dev/quarto-cli/pull/5291
